keys/DO_NOT_USE_IN_PRODUCTION

```diff
@@ -0,0 +1,3 @@
+Those keys are distributed as a convenience/demo.
```

Comment:
This could also be named `README.md` so GitHub shows it when looking at the directory. Could have both files really.

Reply:
Good idea, I will add those.